### DISCUSSION TITLE
Is there a way to make search not go away when it doesnt  encounter the word? allow me to erase and type again

### DISCUSSION THREAD

**Discussion by Raagh** [LazyVim/LazyVim#2771] [Q&A] [answered]
![image](https://github.com/LazyVim/LazyVim/assets/8405459/4afc5b2c-6998-4669-bb9b-30f02e97d126)

if I type a key that is not present then searchbox just disappears and I have to type everything again, pretty annoying

**Comment by dpetka2001** [comment, marked as answer]
Put the following in a file in `~/.config/nvim/lua/plugins/` directory
```lua
-- ~/.config/nvim/lua/plugins/flash.lua
return {
  {
    "folke/flash.nvim",
    opts = {
      modes = {
        -- Disable labels for regular search with `/`
        search = {
          enabled = false,
        },
      },
    },
  },
}
```
